examples/blueprint-construct/index.ts

```diff
@@ -268,5 +269,21 @@ export default class BlueprintConstruct {
             .teams(...teams, new blueprints.EmrEksTeam(dataTeam), new blueprints.BatchEksTeam(batchTeam))
             .enableControlPlaneLogTypes(blueprints.ControlPlaneLogType.API)
             .build(scope, blueprintID, props);
-    }
+
+
+    //     This code test is used to test CodeStar connection ARN, but since it needs
```

Comment:
@tsahiduek this code can stay uncommented if it is moved over to the examples/examples.ts. This file is not used for tests, but you can run it with `npm run examples <pattern-name> deploy`.

Reply:
@tsahiduek as I mentioned before, this code cannot stay here, as it will be attempted to execute for e2e tests. As I mentioned in my comment above, please move it to examples/examples.ts

Hope it makes sense, let me know if you have questions.